src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,17 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "countries")
+
 public class Country implements Cloneable {
```

Comment:
The Cloneable interface is implemented here, but there is no actual need for cloning in the provided context. Unless there is a specific requirement for the Country objects to be cloneable, it would be better to remove the Cloneable interface and the clone method to adhere to the checklist item that states not to add unnecessary code.